cmd/checkcommits/checkcommits.go

```diff
@@ -115,6 +120,17 @@ func checkCommitSubject(config *CommitConfig, commit *Commit) error {
 		return fmt.Errorf("Commit %v: pure whitespace subject", commit.hash)
 	}
 
+	lcSubject := strings.ToLower(subject)
+
+	if strings.HasPrefix(lcSubject, revertPrefix) {
```

Comment:
with this change we are forcing  revert commits to have the `revert` prefix, right?  should this be documented?

Reply:
Agreed. I'll try to find a good location and raise a doc change PR for the main repo...

Reply:
`git revert` automatically adds that, @devimc.  Are you thinking about partial reverts?

Reply:
just in case someone edit it and remove the revert prefix